packages/model-viewer/src/test/features/scene-graph/nodes/primitive-node-spec.ts

```diff
@@ -41,6 +45,28 @@ const findPrimitivesWithVariant = (model: Model, variantName: string) => {
 };
 
 suite('scene-graph/model/mesh-primitives', () => {
+  suite('Primitive with default material', () => {
+    let element: ModelViewerElement;
+    setup(async () => {
+      element = new ModelViewerElement();
+      element.src = KHRONOS_TRIANGLE_GLB_PATH;
+      document.body.insertBefore(element, document.body.firstChild);
+      await waitForEvent(element, 'load');
+    });
+
+    teardown(() => {
+      document.body.removeChild(element);
+    });
+
+    test('has a default material', async () => {
+      const model = element.model!;
+      expect(model[$primitives].length).to.equal(1);
+      expect(model.materials.length).to.equal(1);
+      expect(model.materials[model[$primitives][0][$initialMaterialIdx]].name)
```

Comment:
Seems like you  could just  put `model.materials[0]` and maybe separately check that  `$initialMaterialIdx  === 0`.

Reply:
Done